### PR TITLE
22 additional monitor properties

### PR DESCRIPTION
**Pull request by PavloS-in6k**

### CODE REVIEW

File: client/src/app/devices/details/monitor-constants.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+export class Port {
+    isSupported: boolean;
+    broken: boolean;
+
+    constructor() {
+        this.isSupported = false;
+        this.broken = false;
+    }
+}
```
Comment (YaroslavP-in6k) [inline client/src/app/devices/details/monitor-constants.ts]: divided into separate files ?

File: api/src/main/java/com/in6k/vault13/entity/Device.java [hunk 2]
```diff
@@ -90,4 +92,12 @@ public String getUpdatedAt() {
...
     }
+
+    public Object getDetails() {
+        return details;
+    }
+
+    public void setDetails(Object details) {
+        this.details = details;
+    }
 }
```
Comment (YaroslavP-in6k) [inline api/src/main/java/com/in6k/vault13/entity/Device.java]: really need new lines between fields ?

File: client/src/app/app.module.ts [hunk 3]
```diff
@@ -16,6 +17,7 @@ import { routing }        from './app.routing';
 import {AppComponent}  from './app.component';
 import {HeaderComponent} from './header.component';
 import {AddDevice} from './devices/add-device.component';
+import {MonitorDetails} from './devices/details/monitor.component';
```
Comment (alexanderko) [inline client/src/app/app.module.ts]: class name and file name should use the same name

Comment (YaroslavP-in6k) [inline client/src/app/app.module.ts]: done

**Pull request merged**